NoahHusain/Truncheons-Flagons
# Leaderboard functionality

Fixed leaderboard to show 0 players for new team and 0 for new team score. Added leaderboard styling and sorting capability

##Steps to Test
1. git fetch --all
2. git checkout anm-branch
3. serve and verify leaderboard shows 0 for new team player count and score
